app/controllers/api/v1/customers_controller.rb

```diff
@@ -25,7 +25,7 @@ def index
 
       def show
         render json: Api::V1::CustomerSerializer.new(
-          CustomersWithBalance.new(customers: customer).query.first,
+          CustomersWithBalance.new(Customer.where(id: params[:id])).query.first,
```

Comment:
Hm, so we already have a customer record and reload the whole think just to add the balance? I'm wondering if it would be easier to have a method on the the customer model. But let's leave that for now because that would have implications on the serializer again because the attribute would always be available.